### PR TITLE
Update looks query

### PR DESCRIPTION
**Pull request by bryans99**
1. Update build to latest SDK packages including Looker components
2. Update webpack builds and babel
3. Replace React class component with functional component
4. General cleanup - removed example code as there are a lot more extension examples available now

### CODE REVIEW

File: react/typescript/looks-query/src/index.tsx [hunk 1]
```diff
@@ -32,18 +32,9 @@ window.addEventListener("DOMContentLoaded", async (event) => {
   const root = document.createElement("div");
```
Comment (jkaster) [inline react/typescript/looks-query/src/index.tsx:32]: semicolon again? Guess that's not a prettier setting in this repo

**Review by jkaster — APPROVED** [review]
Looks good. Minor questions.

File: react/typescript/looks-query/src/demo/LookList.tsx [hunk 2]
```diff
@@ -22,39 +22,45 @@
  * THE SOFTWARE.
  */
 
-import React from 'react'
-import {ILook} from '@looker/sdk'
-import {List, Heading, Box, ListItem, Link, Text} from '@looker/components'
+import React from "react";
+import { ILook } from "@looker/sdk";
+import { List, Heading, Box, ListItem, Link, Text } from "@looker/components";
```
[the remark this answers is not in the document]
Comment (bryans99) [reply react/typescript/looks-query/src/demo/LookList.tsx:27]: There is some prettier linting issues here. On my list to fix

File: react/typescript/looks-query/src/index.tsx (hunk 1 above)
Comment (bryans99) [reply react/typescript/looks-query/src/index.tsx:32]: Yep - on the list

**Pull request merged**